src/Module/Diagnostic/ItemSource.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Friendica\Module;
+namespace Friendica\Module\Diagnostic;
```

Comment:
This class should be in `Friendica\Module\Admin` because of the `is_site_admin()` `requirement.